godtools/App/Share/Data/CruOktaAuthentication/CruOktaAuthentication+GetAuthenticationInstance.swift

```diff
@@ -1,32 +1,32 @@
 //
-//  OktaAuthenticationConfiguration.swift
+//  CruOktaAuthentication+GetAuthenticationInstance.swift
 //  godtools
 //
-//  Created by Levi Eggert on 12/2/21.
-//  Copyright © 2021 Cru. All rights reserved.
+//  Created by Levi Eggert on 10/24/22.
+//  Copyright © 2022 Cru. All rights reserved.
 //
 
 import Foundation
 import OktaAuthentication
 
-class OktaAuthenticationConfiguration {
+extension CruOktaAuthentication {
     
-    required init() {
+    static func getNewAuthenticationInstance(appBuild: AppBuild) -> CruOktaAuthentication {
         
-    }
-    
-    func configureAndCreateNewOktaAuthentication(appBuild: AppBuild) -> CruOktaAuthentication {
-                
         let signInPath: String = "auth"
         let signOutPath: String = "auth/logout"
         let redirectBaseUrl: String
         
+        // TODO: Change back redirectBaseUrl. ~Levi
+        /*
         if appBuild.isDebug {
             redirectBaseUrl = "org.cru.godtools.debug"
```

Comment:
@rachaelblue I'm going to create a ticket for this TODO.  For some reason ```org.cru.godtools.debug``` stopped working.